.env.sample

```diff
@@ -0,0 +1,22 @@
+#DEBUG=False
```

Comment:
There are a couple of environment attributes defined in the docker-compose.yaml file, inside both web and database services. Can those be moved into the .env too?

Reply:
Good question.  The master branch currently has…

```
      - ALLOWED_HOSTS=*
      - DEBUG=True
      - DATABASE_URL=postgresql://admin:admin_pw@database/admin
      - FEEDBACK_EMAIL=office-hours-devs@umich.edu
```

As I was working with the app in local dev., I had to set some values there for running the Twilio tests.  I was thinking at the time that they should go in a config file as Matt does here.  I think the values listed above could also go there.